ethereum/contracts/CappedMintableToken.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity >=0.4.22 <0.6.0;
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+
+contract CappedMintableToken is ERC20Mintable {
```

Comment:
It is worth inheriting from `ERC20Detailed`. It will make sure that we have all the information we would expect an `ERC20` to have i.e. `name` and `symbol`